README.md

```diff
@@ -62,19 +61,19 @@ Each component takes various options, some of which are required.
 
 Each `nav-item` must have an `index`, which is an integer, passed in. This allows the selectedIndex to be calculated.
 
-`fixed (optional)`
+`isFixed (optional)`
 
-`nav-trigger-container` accepts a parameter `fixed`, which is a boolean. If you set `fixed` to true, the header will become fixed, so it will be visible regardless of where you scroll on the page.
+`nav-trigger-container` accepts a parameter `isFixed`, which is a boolean. If you set `isFixed` to true, the header will become fixed, so it will be visible regardless of where you scroll on the page.
 
-- Note: when `fixed` is set to true, a scroll event listener is applied to the window. This will set a class of `fixed-and-scrolled` to true whenever the window has been scrolled down. This allows for setting the background color of the navbar to another color, if you want, so that it is easier to see when fixed and scrolled over the page content.
+- Note: when `isFixed` is set to true, a scroll event listener is applied to the window. This will set a class of `is-fixed-and-scrolled` to true whenever the window has been scrolled down. This allows for setting the background color of the navbar to another color, if you want, so that it is easier to see when fixed and scrolled over the page content.
 
 `multiColor (optional)`
 
 `nav-container` accepts a parameter `multiColor`, which is a boolean. You can set `multiColor` to true, which will enable class names like `color-1`, `color-2`, `color-3` etc. on the nav-marker, so you can specify different colors for the indicator.
 
 `totalNavItems (required)`
 
-`nav-container` accepts a parameter `totalNavItems`, which is an integer. This is a number indicating how many nav items you have. It allows the width of the marker to be calculated dynamically.
+`nav-container` accepts a parameter `totalNavItems`, which is an integer. This is a number indicating how many nav items you have. It allows the width of the marker to be calculated dynamically. **Be sure to set `$n-nav-items` in your `app.scss` file (or wherever your Sass variables are stored) as well.**
```

Comment:
@kyleshevlin do we still need to set both variables here? `totalNavItems` and override the SASS variable? I didn't think about this until now, but what about people not using SASS? They won't be able to set a number now. We may need to put it back to the hack with multiple classes 😞